fedoratagger/frontend/templates/master.mak

```diff
@@ -44,5 +44,27 @@
 			<a href="https://github.com/fedora-infra/fedora-tagger/issues">
 			the GitHub issues tracker</a>.
 		</div>
+
+    % if 'FEDMENU_URL' in g.config:
```

Comment:
> How does mako react if you try to access an un-defined variable?

http://docs.makotemplates.org/en/latest/runtime.html#context-variables

> In other words, should we check for FEDMENU_DATA_URL as well?

No, only because we don't do that in other templates in other apps.  So far I've just been assuming that if the admin has configured ``FEDMENU_URL`` then we can expect them to have configured ``FEDMENU_DATA_URL`` also.